target/linux/rockchip/patches-6.6/400-1-arm64-dts-rockchip-Add-common-definitions-for-NanoPi.patch

```diff
@@ -0,0 +1,1573 @@
+From 93be10e4e180eeac9b6a57a4363e08c35bee3af9 Mon Sep 17 00:00:00 2001
+From: Sebastian Kropatsch <seb-dev@mail.de>
```

Comment:
Is someone working on getting these patches accepted upstream?
I can not find them in linux-next.

Reply:
> Is someone working on getting these patches accepted upstream?
> I can not find them in linux-next.

have not been merged yet, they are being reviewed https://lore.kernel.org/all/?q=nanopi-r6

Reply:
The patches were send about 2.5 months ago. Do you know if a updated version will be send soon?

Reply:
I don't have that info, if is strictly a  requirement that the patches be in the kernel tree I can drop `rockchip: fixes and improve for NanoPi R6 series` and `rockchip: rework LED configurations for the NanoPi R6C/R6S`  for later if they get accepted. is OK that?

Reply:
The patches are not looking so bad, I applied your PR.

Please send a new PR when these patches are getting merged upstream.